tests/test_workers/test_tasks.py

```diff
@@ -77,16 +78,26 @@ def test_fetch_gomod_source(
             1, sample_deps_replace, sample_env_vars, 'gomod', [sample_package])
 
     if auto_detect:
-        mock_path_exists.assert_called_once_with('/tmp/cachito-archives/bundles/temp/1/app/go.mod')
+        mock_path_exists.assert_called_once_with(
+            os.path.join(
+                tempfile.gettempdir(), 'cachito-archives/bundles/temp/1/app/go.mod'
+            )
+        )
         if contains_go_mod:
             mock_resolve_gomod.assert_called_once_with(
-                '/tmp/cachito-archives/bundles/temp/1/app', mock_request, dep_replacements,
+                os.path.join(
+                    tempfile.gettempdir(), 'cachito-archives/bundles/temp/1/app'
```

Comment:
Could you assign the value of `os.path.join(tempfile.gettempdir(), 'cachito-archives/bundles/temp/1/app')` to a variable to not duplicate this below?

Reply:
resolved in https://github.com/release-engineering/cachito/pull/97/commits/d08c6066970250e2045c529acd4c1f37f3847c0b